src/test/java/org/broadinstitute/hellbender/utils/SimpleIntervalUnitTest.java

```diff
@@ -208,6 +208,25 @@ public void testOverlapWithMargin( final SimpleInterval firstInterval, final Sim
                 "overlap() returned incorrect result for intervals " + firstInterval + " and " + secondInterval);
     }
 
+    @DataProvider()
+    private Object[][] overlapsWithMarginExpectingException() {
+        final SimpleInterval standardInterval = new SimpleInterval("1", 10, 20);
+        final SimpleInterval middleInterval = new SimpleInterval("1", 100, 200);
+
+        return new Object[][] {
+                { standardInterval, new SimpleInterval("2", 10, 20), -100 },
```

Comment:
This seems like a lot of tests for a pretty simple feature, It's unclear to me what all these cases are, and if they're all needed.